lib/db/db.go

```diff
@@ -25,6 +27,28 @@ type storeWrapper struct {
 	*sql.DB
 }
 
+func (s *storeWrapper) ExecContext(ctx context.Context, query string, args ...any) (sql.Result, error) {
```

Comment:
We'll need this later for [Databricks](https://docs.databricks.com/en/dev-tools/go-sql-driver.html#manage-files-in-unity-catalog-volumes:~:text=NewContextWithStagingInfo).